### PR TITLE
Update communicating to a node in the RPC page in Lisk docs

### LINKED ISSUE #1754: Update communicating to a lisk node via RPC page
### Description
In the API client section, add in a short description/example showing the options that can be deployed to enable certain modes of the client.

#### Affected branch / URL
development

#### Page location
understand-blockchain/sdk/rpc.adoc

### PR DESCRIPTION
**Pull request by chris529**
Closes #1754

### CODE REVIEW

File: docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc [hunk 1]
```diff
@@ -165,6 +165,39 @@ The three modes of communication i.e. IPC, WS, and HTTP allow sApps to communica
     * HTTP requests establish a uni-directional connection to the server and close it once a response is received.
     * Since an event emits data in real time, a subscription can only work if the receiver's connection is persistent. This is not possible with an HTTP connection, hence it cannot be used to subscribe to events.
 
+=== Enabling IPC, WebSocket, and HTTP APIs
+
+Examples of the different options that can be deployed to enable certain modes of the client can be seen below.
+There are two methods that can be used to expose the node APIs via IPC, WS, and HTTP.
+
+There are two ways to expose the node APIs via IPC, WS, and HTTP.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc]: Redundant 
```suggestion
```

File: docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc [hunk 2]
```diff
@@ -165,6 +165,39 @@ The three modes of communication i.e. IPC, WS, and HTTP allow sApps to communica
...
+
+- `--api-ipc`: Enable API over IPC
+- `--api-ws`: Enable API over WS
+- `--api-http`: Enable API over HTTP
+- `--api-host=127.0.0.1`: IP over which the API is exposed. Use `0.0.0.0` to expose the API on all available ethernet IP instances
+- `--api-port=7887`: Port on which the API is exposed
+
+2. This can also be achieved by adding the RPC configuration option to your Lisk node's configuration file as shown below.
+To enforce the config changes `--overwrite-config` flag must be passed when starting the application.
+To supply a custom file use the `--config=</path/to/custom/config>` flag:
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc]:
```suggestion
To supply a custom config file, use the `--config=</path/to/custom/config>` flag:
```

File: docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc [hunk 3]
```diff
@@ -165,6 +165,39 @@ The three modes of communication i.e. IPC, WS, and HTTP allow sApps to communica
...
+
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc]:
```suggestion
+
```

**Review by TalhaMaliktz — APPROVED** [review]
LGTM! just some minor remarks.

File: docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc [hunk 4]
```diff
@@ -165,6 +165,38 @@ The three modes of communication i.e. IPC, WS, and HTTP allow sApps to communica
...
+- `--api-ws`: Enable API over WS
+- `--api-http`: Enable API over HTTP
+- `--api-host=127.0.0.1`: IP over which the API is exposed. Use `0.0.0.0` to expose the API on all available ethernet IP instances
+- `--api-port=7887`: Port on which the API is exposed
+
+2. This can also be achieved by adding the RPC configuration option to your Lisk node's configuration file as shown below.
+To enforce the config changes `--overwrite-config` flag must be passed when starting the application.
+To supply a custom config file, use the `--config=</path/to/custom/config>` flag:
++
+[source,jsonc]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc]: One other correction
```suggestion
[source,json]
```

File: docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc [hunk 5]
```diff
@@ -165,6 +165,38 @@ The three modes of communication i.e. IPC, WS, and HTTP allow sApps to communica
     * HTTP requests establish a uni-directional connection to the server and close it once a response is received.
     * Since an event emits data in real time, a subscription can only work if the receiver's connection is persistent. This is not possible with an HTTP connection, hence it cannot be used to subscribe to events.
 
+=== Enabling IPC, WebSocket, and HTTP APIs
+
+Examples of the different options that can be deployed to enable certain modes of the client can be seen below.
```
Comment (nagdahimanshu) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/rpc.adoc]: May be we can update this to:
```suggestion
Examples of the different options that can be used to enable certain modes of the client can be seen below.
```
or
```suggestion
Below are examples of the different options that can be used to enable certain client modes.
```

**Review by nagdahimanshu — APPROVED** [review]

**Pull request merged**